src/strategies/staked-defi-balance/examples.json

```diff
@@ -1,48 +1,118 @@
-                "name": "id",
-                "type": "address"
-              },
-              {
-                "internalType": "address",
-                "name": "staker",
-                "type": "address"
-              }
-            ],
-            "name": "stakeOf",
-            "outputs": [
-              {
-                "internalType": "uint256",
-                "name": "",
-                "type": "uint256"
-              }
-            ],
-            "stateMutability": "view",
-            "type": "function"
-          }
-        ]
-      }
-    },
-    "network": "42161",
-    "addresses": [
-      "0x4c4180bf5ec78af9025bdd935ed69e29c2f6cbae",
-      "0x5785530b6f0ea72b0dba474d55b43e1af1182cad",
-      "0x2ed211293b1db5e7d177e2711bea83700e45b150"
-    ],
-    "snapshot": 74259791
-  }
+    {
+        "name": "Staked Defi Balance",
+        "strategy": {
+            "name": "staked-defi-balance",
+            "params": [
+                {
+                    "network": "42161",
+                    "snapshot": 85322593,
+                    "tokenContractAddress": "0xe685d3CC0be48BD59082eDe30C3b64CbFc0326e2",
+                    "symbol": "cFRM",
+                    "decimals": 18,
+                    "minStakedBalance": "500000000000000000000000",
+                    "stakingPoolContractAddress": "0xb4927895cbee88e651e0582893051b3b0f8d7db8",
+                    "methodABI": [
+                        {
+                            "inputs": [
+                                {
+                                    "internalType": "address",
+                                    "name": "id",
+                                    "type": "address"
+                                },
+                                {
+                                    "internalType": "address",
+                                    "name": "staker",
+                                    "type": "address"
+                                }
+                            ],
+                            "name": "stakeOf",
+                            "outputs": [
+                                {
+                                    "internalType": "uint256",
+                                    "name": "",
+                                    "type": "uint256"
+                                }
+                            ],
+                            "stateMutability": "view",
+                            "type": "function"
+                        }
+                    ]
+                },
+                {
+                    "network": "56",
+                    "snapshot": 27877333,
+                    "tokenContractAddress": "0xaf329a957653675613D0D98f49fc93326AeB36Fc",
+                    "symbol": "cFRM",
+                    "decimals": 18,
+                    "minStakedBalance": "500000000000000000000000",
+                    "stakingPoolContractAddress": "0x35e15ff9ebb37d8c7a413fd85bad515396dc8008",
```

Comment:
Instead of passing multiple items like this, better to use multiple strategies in your space :)

Reply:
Thanks for the suggestion @ChaituVR but currently, there aren't strategies that can handle the use case. In the future, I plan to create `staked-defi-balance-traditional-staking` and `staked-defi-balance-open-staking` strategies then I can pass strategies here. 

Until that is done, we need this use case where params can hold the info to run multi-chain and multi-contract checks from the same strategy. 🙌

Reply:
Hi @taha-abbasi This could increase the server load. and others will refer to this strategy and ask us to allow passing multiple strategies into a single strategy, ideally, it should ideally accept one `network` and one `snapshot` You can use a strategy multiple times in your space so would be great if you can modify like that, thanks for understanding 🙏

Reply:
Hi @ChaituVR Please see the updates that include a new strategy per your suggestions:

Release Notes
=============

staked-defi-balance v1.2.6
--------------------------

### Improvements

-   No need to provide multiple params anymore: The strategy has been optimized to work with the new multi-staked-defi-balance strategy, reducing the complexity of the configuration.
-   Removed the need to provide methodABI: Users don't need to provide methodABI in the configuration anymore, making it simpler to use.
-   Better integration with multi-staked-defi-balance: The staked-defi-balance strategy has been improved to work seamlessly with the new multi-staked-defi-balance strategy.

multi-staked-defi-balance v1.0.0
--------------------------------

### New Features

-   Multi-strategy support: multi-staked-defi-balance allows users to pass multiple strategies instead of passing complex params with methodABIs, making it more efficient and simpler to use.
-   Cumulative staked balance calculation: The strategy calculates the cumulative staked balance across multiple staking contracts and networks for each user.
-   Flexible configuration: Users can configure multiple staking contracts across different networks and staking types (open and standard) in a single strategy.

### Parameters

-   `symbol`: The symbol for the cumulative staked balance.
-   `cumulativeMinimumStakedBalance`: The minimum cumulative staked balance required for a user to pass the strategy.
-   `strategies`: An array of staked-defi-balance strategy objects, each representing a different staking contract configuration.

For more information about the parameters and usage, please refer to the README.md.

### Compatibility

-   The multi-staked-defi-balance strategy is fully compatible with the staked-defi-balance strategy v1.2.6, and it's recommended to use them together for a better user experience.

We hope that these updates make the staked-defi-balance and multi-staked-defi-balance strategies more efficient and easier to use. As always, contributions, suggestions, and bug reports are welcome! Please submit them through our GitHub repository.


Test Results
=========
`staked-defi-balance`
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/11986835/236969603-d30cc38a-ad61-4fdb-8d7f-7324e4172247.png">

`staked-defi-balance with 500 addresses`
<img width="952" alt="image" src="https://user-images.githubusercontent.com/11986835/236969703-bd0bcefe-6b04-4456-8583-914049c158e9.png">

`multi-staked-defi-balance`
<img width="983" alt="image" src="https://user-images.githubusercontent.com/11986835/236969891-b8c9c91a-0a11-4249-be6e-35830bd209b9.png">

`multi-staked-defi-balance with 500 addresses`
<img width="903" alt="image" src="https://user-images.githubusercontent.com/11986835/236970038-2aa2fd31-f4f8-4842-b9db-c957dffc1b6b.png">


I hope this satisfies the feedback, looking forward to hearing from you and the merge.